zosia_site/locals.tf

```diff
@@ -1,6 +1,7 @@
 locals {
-  project_id = ""
-  region     = "europe-central2"
+  project_id        = ""
```

Comment:
You can get this from the [project datasource](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/project), but it's better to provide it explicitly to ensure you're running it on the correct project. Therefore, I suggest adding a comment like "< update this >".

Reply:
Done